pkg/replicate/replicater.go

```diff
@@ -149,6 +150,11 @@ func RunReplicate(
 		Help: "The Duration of replication runs split by success and error.",
 	}, []string{"result"})
 
+	fetcher, err := thanosblock.NewMetaFetcher(logger, 32, fromBkt, "", reg)
+	if err != nil {
+		return errors.Errorf("create meta fetcher with bucket %v: %w", fromBkt, err)
```

Comment:
errors.Wrap is our way to wrap errros (:

Reply:
:+1: and this will become an error on `go vet` with 1.14